src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -220,17 +163,16 @@ public PluginCompatReport testPlugins()
 
         PluginCompatReport report = PluginCompatReport.fromXml(config.reportFile);
 
-        SortedSet<MavenCoordinates> testedCores = config.getWar() == null ? generateCoreCoordinatesToTest(data, report) : coreVersionFromWAR(data);
+        MavenCoordinates coreCoordinates = new MavenCoordinates("org.jenkins-ci.main", "jenkins-war", data.core.version);
```

Comment:
Simplifying the code to only handle the WAR case. While I was here I also updated the group ID and artifact ID to match the actual ones of the Jenkins WAR. The previous ones were incorrect. This doesn't affect the logic, but is only used for display purposes during logging. Using the correct group ID and artifact ID makes for a less confusing experience when reading the logs.